.github/workflows/pull_request.yml

```diff
@@ -55,6 +55,10 @@ jobs:
           pytest --collect-only --disable-pytest-warnings -n 2 -m pre_upgrade tests/upgrades/
           pytest --collect-only --disable-pytest-warnings -n 2 -m post_upgrade tests/upgrades/
 
+      - name: Evaluate Fixture Use
+        run: |
+          pytest --setup-plan tests/foreman/
```

Comment:
We need to run against `tests/upgrade` too, with `-m pre_upgrade` and `-m post_upgrade` like in the above collection blocks. 

You can actually just add `--setup-plan` to the above blocks, because it'll collect and dump the plan (validating fixture scope in the process).